ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.hbs

```diff
@@ -28,82 +28,101 @@
 
   <page.body>
     {{#if @model.hosts}}
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            {{#if @model.hostSet.isStatic}}
-              <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            {{else}}
-              <row.headerCell>{{t 'form.external_name.label'}}</row.headerCell>
-            {{/if}}
-            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            {{#if @model.hostSet.isStatic}}
-              <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-            {{/if}}
-          </header.row>
-        </table.header>
-
-        <table.body as |body|>
-          {{#each @model.hosts as |host|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if host.isStatic}}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
-                    @model={{host.id}}
-                  >
-                    {{host.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{! Link to a different subroute for dynamic/plugin hosts }}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog.host-sets.host-set.hosts.host'
-                    @model={{host.id}}
-                  >
-                    {{host.displayName}}
-                  </LinkTo>
-                {{/if}}
-                <p>{{host.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                <HostCatalogTypeBadge @model={{host}} />
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{host.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
+      {{#if (get @model.hostSet 'isStatic')}}
+        <Hds::Table
+          @valign='middle'
```

Comment:
None blocking: Perhaps nit picky, but can we put the `@valign` attribute the last one? so `@model` and `@columns` are first. This is how we have been defining HDS table attributes in the previous work and personally, I think it is good practice to have more important attributes first. Totally debatable and open to it.

Reply:
@calcaide absolutely agreed